src/NuGet.Services.EndToEnd/Support/Clients/V2V3SearchClient.cs

```diff
@@ -322,7 +337,12 @@ public class AutocompleteResponse
             public string Index { get; set; }
             public long TotalHits { get; set; }
             public DateTimeOffset LastReopen { get; set; }
+        }
 
+        public class SearchService
```

Comment:
I think the `set;` is only ever called when the instance is constructed, so we don't need a public setter.